python/dgl/transforms/module.py

```diff
@@ -1397,6 +1397,7 @@ class NodeShuffle(BaseTransform):
             [ 7.,  8.]])
     """
     def __call__(self, g):
+        g = g.clone()
```

Comment:
Will this be super expensive if we clone a graph?

Reply:
I think it is ok as this is typically applied to graphs of 100-1000 nodes.